doc/contributing/coding-conventions.chapter.md

```diff
@@ -196,7 +196,23 @@
   As an exception, an explicit conditional expression with null can be used when fixing a important bug without triggering a mass rebuild.
   If this is done a follow up pull request _should_ be created to change the code to `lib.optional(s)`.
 
-- Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.
+- The order of elements in the argument set must follow the convention below:
+
+  01. `lib`;
+  02. `stdenv`;
+  03. builders (`buildPythonPackage`, `mkDerivation`, `buildGoModule`);
+  04. fetchers (`fetchurl`, `fetchpatch`, `fetchFromGitHub`);
+  05. `buildPackages`;
```

Comment:
other sets should also be mentioned here, e.g. `pkgsBuildBuild` and `pkgs`

Reply:
Nice.